### DISCUSSION TITLE
Min and Max Time

### DISCUSSION THREAD

**Discussion by sureshfinekube** [dmuy/MDTimePicker#17] [Q&A] [answered]
is possible to set Min Time : 08:00PM and Max Time : 08:00AM ? I need to set max time as current time + 12 Hours. after setting like this, time picker is disabled.when the max time is greater than 12AM. how to solve this problem?

**Comment by dmuy** [comment, marked as answer]
Hello @sureshfinekube,

So, the min & max functionality is actually quite a "controversial" topic (I should say) because to be able to validate the min & max time, I use a date - the current date to be exact. For your use case, it crosses unto the next day which is currently not possible.

Currently, I have no idea on how to do this though - how to validate the min & max with this kind of requirement.

**Reply by sureshfinekube** [reply]
Thank you for valuable reply.i removed max and min parameters from MDTimePicker. and I used [moment.js](https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.2/moment.js) for [calculating difference between time](http://jsfiddle.net/nil4you/gs69Lv5x/). if hours(or)minutes return the negative values considered as tomorrow time and both are negative it will be today time. add more features to MDTimePicker. it is very user-friendly. once again thank you for your immediate reply.

**Reply by dmuy** [reply]
The thing is, the time picker has no indicator if I'm picking tomorrow's time or yesterday's - it's just for the current-day-setting type of thing. So, I'm having a hard time how to implement this kind of feature.
